README.md

```diff
@@ -34,39 +34,16 @@ using the Docker image.
 
 ### Prerequisites
 
-1. Beginning with Swarm 0.4 golang 1.4.x or later is required for building Swarm. 
-Refer to the [Go installation page](https://golang.org/doc/install#install)
-to download and install the golang 1.4.x or later package.
-> **Note**: On Ubuntu 14.04, the `apt-get` repositories install golang 1.2.1 version by
-> default. So, do not use `apt-get` but install golang 1.4.x manually using the
-> instructions provided on the Go site.
-
-2. Install [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).
-
-3. Install [godep](https://github.com/tools/godep).
+Install [Docker](http://docs.docker.com/installation/)
 
 ### Clone and build Swarm
 
-> **Note** `GOPATH` should be set when install godep in above step.
-
-Install the `swarm` binary in the `$GOPATH/bin` directory. An easy way to do this 
-is using the `go get` command.
-
-```bash
-$ go get github.com/docker/swarm
-```
-
-You can also do this manually using the following commands:
-
 ```bash
-$ mkdir -p $GOPATH/src/github.com/docker/
-$ cd $GOPATH/src/github.com/docker/
-$ git clone https://github.com/docker/swarm
-$ cd swarm
-$ $GOPATH/bin/godep go install .
+$ git clone git@github.com:docker/swarm && cd swarm
+$ make binary
```

Comment:
That seems really strange to me since this is one of the primary use cases for Docker (building things independent from the host), and this is a docker project!

That said, it's not so much a requirement just the "suggested path for getting started" to make it easy for contributors.